utils/blob_utils.py

```diff
@@ -0,0 +1,17 @@
+from azure.storage.blob import BlockBlobService, ContentSettings
+
+class BlobStorage():
```

Comment:
@samiyaakhtar I think this is a good start. Eventually this code and the blob storage code the functions use should be the same. Ideally we may want to fully encapsulate "storage". Meaning that blob store specific operations live here and clients  (like training.py) don't "know" about blob storage. All the client knows is a save something or I retrieve something. This makes functionally testing a client (like training.py) somewhat easier. No action required now but just thinking about a possible future

Reply:
That sounds like a good idea, more abstraction for the training script should eventually be put into place